SU-SWS/saa_victoria_subtheme
# Adapt 687  improve template inheritance

# READY BUT NEEDS TO BE MERGED AT THE RIGHT TIME
- evaluating option to provide blocks to extend from stanford_basic
- needs to be merged WHEN/after Stanford Basic theme has been updated to use twig blocks, so these changes will work.

# Summary
- Update this subtheme to @extend from stanford_basic better, assuming we add blocks into stanford_basic templates.
- Background on this change in stanford basic is here: https://stanfordits.atlassian.net/browse/D8CORE-2677

# Review By (Date)
- future date

# Urgency
- low, has related accessibility issues it would HELP with

# Steps to Test

1. Pull down the branch for this pr for the saa_victoria_theme
2. Check out this branch of stanford_basic: https://github.com/SU-SWS/stanford_basic/pull/183
3. Clear the site cache
4. See that the page sections render properly. There shouldn't be visual changes. Review code in this pr and in the pr for stanford_basic: https://github.com/SU-SWS/stanford_basic/pull/183

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-687
- this ticket prompted this d8core ticket for updating stanford basic: https://stanfordits.atlassian.net/browse/D8CORE-2677
- Other PRs - this one in stanford basic: https://github.com/SU-SWS/stanford_basic/pull/183 
- This pr needs to be merged **WHEN/after Stanford Basic theme has been updated** (see pr above) to use twig blocks, so these changes will work.
- Once the release of Stanford Basic is ready, then **this branch of the victoria theme needs an updated package.json file with that stanford_basic version as a dependency**. @sherakama started this requirement here: https://github.com/SU-SWS/saa_victoria_subtheme/pull/122/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R10
- cc @cjwest
